<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClinicInternalPayment</class>
 <widget class="QWidget" name="ClinicInternalPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>门诊内部缴款</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="printButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="previewButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>预览</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="excelButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Excel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>日期：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>13</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>金额</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>西药费</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>中成药费</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>挂号费A</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>检查费D</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>治疗费E</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>护理费F</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>手术费G</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>化验费H</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>材料费</string>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>诊查费C</string>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string>其他费</string>
          </property>
         </item>
         <item row="11" column="0">
          <property name="text">
           <string>床位费</string>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string>一般诊疗费</string>
          </property>
         </item>
         <item row="13" column="0">
          <property name="text">
           <string>合计</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
